src/cmdstan-guide/pathfinder_config.Rmd

```diff
@@ -0,0 +1,162 @@
+# Pathfinder Method for Approximate Bayesian Inference
+
+The CmdStan method `pathfinder` uses the Pathfinder algorithm of @zhang_pathfinder:2022.
+Pathfinder is a variational method for approximately
+sampling from differentiable log densities.
+Starting from a random
+initialization, Pathfinder locates normal approximations to the target
+density along a quasi-Newton optimization path, with local covariance
+estimated using the inverse Hessian estimates produced by the
+L-BFGS optimizer.  Pathfinder returns draws from the Gaussian approximation
+with the lowest estimated Kullback-Leibler (KL) divergence to the true
+posterior.
+
+The full set of configuration options available for the `pathfinder` method is
+reported at the beginning of the pathfinder output file as CSV comments.
+When the example model `bernoulli.stan` is run with `method=pathfinder`
+via the command line with all default arguments,
+the resulting Stan CSV file header comments show the complete set
+of default configuration options:
+```
+method = pathfinder
+  pathfinder
+    init_alpha = 0.001 (Default)
+    tol_obj = 9.9999999999999998e-13 (Default)
+    tol_rel_obj = 10000 (Default)
+    tol_grad = 1e-08 (Default)
+    tol_rel_grad = 10000000 (Default)
+    tol_param = 1e-08 (Default)
+    history_size = 5 (Default)
+    num_psis_draws = 1000 (Default)
+    num_paths = 4 (Default)
+    max_lbfgs_iters = 1000 (Default)
+    num_draws = 1000 (Default)
+    num_elbo_draws = 25 (Default)
+```
+
+The arguments `init_alpha` through `history_size` are used by the L-BFGS algorithm.
```

Comment:
These are all used by Pathfinder---the first lot get passed through to the embedded use of L-BFGS.